bundles/binding/org.openhab.binding.temperlan/src/main/java/org/openhab/binding/temperlan/internal/TemperlanBinding.java

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.temperlan.internal;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openhab.binding.temperlan.TemperlanBindingProvider;
+import org.openhab.binding.temperlan.internal.TemperlanBinding;
+import org.openhab.binding.temperlan.internal.TemperlanBindingConfig;
+import org.openhab.binding.temperlan.internal.TemperlanBindingConfig.BindingType;
+import org.openhab.binding.temperlan.internal.hardware.TemperlanProxy;
+import org.openhab.binding.temperlan.internal.hardware.TemperlanState;
+
+import org.apache.commons.lang.StringUtils;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+	
+
+/**
+ * Implement this class if you are going create an actively polling service
+ * like querying a Website/Device.
+ * 
+ * @author snoerenberg
+ * @since 1.0
+ */
+public class TemperlanBinding extends AbstractActiveBinding<TemperlanBindingProvider> implements ManagedService {
+
+	public static final String CONFIG_KEY_HOST = "host";
+	public static final long DEFAULT_REFRESH_INTERVAL = 60000;
+	public static final String DEFAULT_DEVICE_UID = "default";
+
+	private static final String BINDING_NAME = "TemperlanBinding";
+
+	private static final Logger logger = LoggerFactory
+			.getLogger(TemperlanBinding.class);

```

Comment:
please run all Java files through the openHAB formatter in Eclipse IDE (120-character line length, etc.)